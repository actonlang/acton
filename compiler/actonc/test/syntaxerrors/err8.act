def f(x):
    y = 2
     z = x+y
    return z
