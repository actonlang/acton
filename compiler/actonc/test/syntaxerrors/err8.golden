Building file test/syntaxerrors/err8.act using temporary scratch directory

ERROR: Error when compiling err8 module: Indentation error
   |
16 |     z = x+y
   |     ^
Too much indentation

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 

